components/clarinet-files/src/project_manifest.rs

```diff
@@ -45,30 +46,155 @@ pub struct ProjectConfigFile {
     cache_dir: Option<String>,
 }
 
-#[derive(Serialize, Debug, Clone)]
+#[derive(Deserialize, Serialize, Debug, Clone)]
 pub struct ProjectManifest {
     pub project: ProjectConfig,
     #[serde(serialize_with = "toml::ser::tables_last")]
+    #[serde(deserialize_with = "contracts_deserializer")]
     pub contracts: BTreeMap<String, ClarityContract>,
     #[serde(rename = "repl")]
     pub repl_settings: repl::Settings,
-    #[serde(skip_serializing)]
-    pub location: FileLocation,
-    #[serde(skip_serializing)]
+    //#[serde(skip_serializing)]
```

Comment:
We'll need to keep the `skip_serializing` directive here.

To get deserialization to work, maybe we need to require that the user provides a location of the manifest? Because this packaged version has everything needed to run a devnet, they can really make this location wherever they want.

Maybe we can default to some OS-specific temp directory in the case that a user doesn't provide a location?